model-optimizer/extensions/middle/SplitConcatPairToInterpolate.py

```diff
@@ -53,7 +66,16 @@ def get_concat_after_split(split: Node) -> Optional[Node]:
 
     dest = split.out_port(0).get_destinations()[0].node
     # The transformation is applicable, only if next node is Concat.
-    return dest if dest.soft_get('type') == 'Concat' else None
+    if dest.soft_get('type') != 'Concat':
+        return
+
+    # The transformation is applicable, only if Split is a unique producer for Concat.
+    dest_inputs = [p.get_connection().get_source().node for _, p in dest.in_ports().items() if not p.disconnected()]
+    names_of_concat_inputs = set([n.name for n in dest_inputs])
+    if len(names_of_concat_inputs) != 1:
```

Comment:
TensorFlow ConcatV2 operation has a separate input (not attribute!) containing axis for concatenation. So in this case the number of unique names for Concat inputs will be equal to 2. https://github.com/tensorflow/tensorflow/blob/master/tensorflow/core/ops/array_ops.cc#L523-L525

Reply:
Fixed.